src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -26,40 +27,47 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   roomUrl,
 }) => {
   const dispatch = useDispatch();
+
+  const schedulePrimaryClasses = classNames({
+    "ScheduleItem--primary": isCurrentEvent,
+  });
+
+  const scheduleItemTimeSectionClasses = classNames(
+    "ScheduleItem__time-section",
+    schedulePrimaryClasses
+  );
+
   return (
-    <div className="schedule-item-container">
-      <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
-        <div>
-          <b>{formatUtcSeconds(event.start_utc_seconds)}</b>
-        </div>
-        <div>
+    <div className="ScheduleItem">
+      <div className={scheduleItemTimeSectionClasses}>
+        <span className="ScheduleItem__event-date">
+          {labelDayRoomSchedule(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
+          {formatUtcSeconds(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
           {formatUtcSeconds(
             event.start_utc_seconds + event.duration_minutes * 60
           )}
-        </div>
+        </span>
       </div>
-      <div className="event-section">
-        <div>
-          <div className={`${isCurrentEvent ? "primary" : ""}`}>
-            <div>
-              <b>{event.name}</b>
-            </div>
-            <div>
-              by <b>{event.host}</b>
-            </div>
-          </div>
-          <div className="event-description">
+      <div className="ScheduleItem__event-section">
+        <div className={schedulePrimaryClasses}>
+          <span className="ScheduleItem__event-name">{event.name}</span>
+          by <span className="ScheduleItem__event-host">{event.host}</span>
+          <span className="ScheduleItem__event-description">
             <RenderMarkdown text={event.description} />
-          </div>
+          </span>
         </div>
         {isCurrentEvent && (
-          <div className="entry-room-button">
+          <div className="ScheduleItem__entry-room-button">
             <a
               onMouseOver={() => dispatch(retainAttendance(true))}
               onMouseOut={() => dispatch(retainAttendance(false))}
-              className="btn room-entry-button"
+              className="btn ScheduleItem__room-entry-button"
               onClick={onRoomEnter}
-              id={`enter-room-from-schedule-event-${event}`}
+              id={`ScheduleItem__enter-room-from-schedule-event-${event}`}
               href={roomUrl}
               target="_blank"
               rel="noopener noreferrer"
```

Comment:
This should be using `externalUrlAdditionalProps`:
```suggestion
              {...externalUrlAdditionalProps}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/78054d17ebf3e5222fe89f80fc682d83b39b96e3